playbooks/setup_jenkins_slave.yml

```diff
@@ -11,8 +11,8 @@
     allow_jenkins_sudo: "{% set v=lookup('env', 'allow_jenkins_sudo') %}{% if v=='' %}True{% else %}{{v|bool}}{% endif %}"
     jenkins_node_exclusive: "{% set v=lookup('env', 'jenkins_node_exclusive') %}{% if v=='' %}True{% else %}{{v|bool}}{% endif %}"
     # These variables are not booleans so can use the standard default filter
-    jenkins_node_labels: "{% set v=lookup('env', 'JENKINS_NODE_LABELS') %}{% if v=='' %}single_use_slave{% else %}{{v}}{% endif %}"
-    jenkins_node_executors: "{% set v=lookup('env', 'JENKINS_NODE_EXECUTORS') %}{% if v=='' %}2{% else %}{{v}}{% endif %}"
+    jenkins_node_labels: "{{ lookup('env', 'JENKINS_NODE_LABELS')|default('single_use_slave', true) }}"
+    jenkins_node_executors: "{{ lookup('env', 'JENKINS_NODE_EXECUTORS')|default('2', true) }}"
```

Comment:
This is not a rebase revert, the [original version had a closing paren in the wrong place](https://github.com/rcbops/rpc-gating/commit/8a67c16d80d5683609e5e50fc12d560c8df77ff6#diff-5be9faccf8f48d89f54f9248ca4de6e3R14), this fixes it.